config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp_ovn_cluster.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - download-cache
+    - bootstrap
+    - configure-network
+    - validate-network
+    - frr
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - neutron-metadata
+    - ovn-bgp-agent
+    - libvirt
+    - nova
+    - telemetry
+  preProvisioned: true
```

Comment:
Do we only support BGP with pre-provisioned nodes? I recall we had issues with IPAM in the last iteration of this work because we couldn't use the same IP address multiple times? 

Is this still the case?

Reply:
I think I never tried the non-preProvisioned option but I don't see why it should not work